### ISSUE TITLE
[Low] [ATAG Briefs] [All] Add reference to standards in summary

### ISSUE THREAD

**Issue by daniel-montalvo** [w3c/wai-intro-atag#37]
Current summary box is:

> This page briefly introduces a standard to help you make your no-code website tool accessible to creators, designers, and end users with disabilities.

"Why Accessibility Matters" section does not have any reference to the word "standard" or to ATAG, but that's deliberate based on historical negative reactions that ATAG in this context tends to throw people off.

Proposal is to leave as-is, given that the summary box already contains the word "standard" and then the last section connects the personas to the standard itself.
